test/renderer.test.ts

```diff
@@ -21,8 +21,8 @@ describe('renderer', () => {
     const { renderScripts } = await getRenderer()
     const result = renderScripts().split('</script>').slice(0, -1).map(s => `${s}</script>`).sort()
     expect(result).to.deep.equal([
-      '<script type="module" src="/entry.mjs" defer></script>',
-      '<script type="module" src="/index.mjs" defer></script>'
+      '<script type="module" src="/entry.mjs" async></script>',
```

Comment:
Are you sure async (parallel execution) is supported with both vite and webpack chunks? Last time checked it wasn't for webpack.